src/plugin/renderer/material_manager.cpp

```diff
@@ -34,10 +34,13 @@ void wmr::MaterialManager::Initialize()
 	wr::Material* internal_material = m_material_pool->GetMaterial( m_default_material_handle.m_id );
 	auto& texture_manager = renderer.GetTextureManager();
 
-	internal_material->SetAlbedo( texture_manager.GetDefaultTexture() );
-	internal_material->SetNormal( texture_manager.GetDefaultTexture() );
-	internal_material->SetMetallic( texture_manager.GetDefaultTexture() );
-	internal_material->SetRoughness( texture_manager.GetDefaultTexture() );
+	internal_material->SetUseConstantAlbedo( true );
+	internal_material->SetUseConstantMetallic( true );
+	internal_material->SetUseConstantRoughness( true );
```

Comment:
No normal map will cause Hybrid to crash

Reply:
I set UseNormalTexture to false. this should work. otherwise, we add an issue for Wisp